airflow/www/templates/airflow/dags.html

```diff
@@ -207,14 +214,21 @@ <h2>DAGs</h2>
 
 {% block tail %}
   {{ super() }}
+  <script src="{{ url_for_asset('utils.js') }}"></script>
   <script src="{{ url_for_asset('d3.min.js') }}"></script>
   <script src="{{ url_for_asset('jquery.dataTables.min.js') }}"></script>
   <script src="{{ url_for_asset('dataTables.bootstrap.min.js') }}"></script>
   <script src="{{ url_for_asset('bootstrap-toggle.min.js') }}"></script>
   <script src="{{ url_for_asset('bootstrap3-typeahead.min.js') }}"></script>
   <script>
 
-      const DAGS_INDEX = "{{ url_for('Airflow.index') }}";
+      const DAGS_INDEX = getMetaValue('base-url-index');
+      const BASE_URL_BLOCKED = getMetaValue('base-url-blocked');
+      const BASE_URL_DAG_STATS = getMetaValue("base-url-dag-stats");
```

Comment:
Given this is inline it won't be compiled: what browsers do/don't support `const`? (I'm way out of date with what is ES "Next" and what is widely supported now)

Reply:
If we accept this patch, in the next change I will extract the Javascript code into separate files to be able to compile it into old browsers - babel. New JS features are too widely accepted in the community to write code in old standards.

Reply:
In addition, your suggestion is not compatible with the code rules adopted in the project. This project follow the Airbnb recommendation
https://github.com/apache/airflow/blob/master/CONTRIBUTING.md#javascript-style-guide
> 2.1 Use const for all of your references; avoid using var. eslint: prefer-const, no-const-assign
https://github.com/airbnb/javascript#references

Reply:
Introduction your suggestion will also add confusion in the code, because the "var" syntax is used in the same file.  The code does not work on older versions of the Internet Explorer. However, there are a lot of reasons why this does not work and the best solution is to extract the code for separate files and transpile all JS using Babel. Transpilation will allow the developer to use new syntax and functionality, but browsers will execute code that conforms to the old standards.
Please note that some of the JS code is currently being transpiled.
https://github.com/apache/airflow/blob/master/airflow/www/package.json#L29-L34
This is not a problem that we want to solve in this PR. The JS code has many issues that have to be gradually solved.

Reply:
Summary: In the current state of the code, we can not worry about unsupported browsers. Support for other browsers can be introduced in a separate PR.

Reply:
I hope that the arguments will be fine for you.